Makefile

```diff
@@ -173,6 +173,7 @@ $(RUNTIME_TEST_SRCS): $(GO_PLUGIN) $(RUNTIME_TEST_PROTO)
 $(EXAMPLE_GWSRCS): ADDITIONAL_GW_FLAGS:=$(ADDITIONAL_GW_FLAGS),grpc_api_configuration=examples/internal/proto/examplepb/unannotated_echo_service.yaml
 $(EXAMPLE_GWSRCS): $(GATEWAY_PLUGIN) $(EXAMPLES)
 	protoc -I $(PROTOC_INC_PATH) -I. -I$(GOOGLEAPIS_DIR) --plugin=$(GATEWAY_PLUGIN) --grpc-gateway_out=logtostderr=true,allow_repeated_fields_in_body=true,$(PKGMAP)$(ADDITIONAL_GW_FLAGS):. $(EXAMPLES)
+	protoc -I $(PROTOC_INC_PATH) -I. -I$(GOOGLEAPIS_DIR) --plugin=$(GATEWAY_PLUGIN) --grpc-gateway_out=logtostderr=true,allow_repeated_fields_in_body=true,paths=source_relative,import_prefix="github.com/grpc-ecosystem/grpc-gateway/v2/",standalone=true$(ADDITIONAL_GW_FLAGS):examples/internal/proto/standalone $(EXAMPLES)
```

Comment:
This generates way too many files :joy:. Lets just choose one of the packages (a simple one) and generate a standalone package for that instead of all of them.

Reply:
What does the `import_prefix` parameter do here? I've never had to use it before.

Reply:
My point to generate all examples was to check that my feature accept all edge cases described in .proto. Looks like it does, so ok, i leave only one example.
The way you can pass external package path is to set it inside package in go_package option, but I didn't want to change code of the examples, so I decide to simulate such behavior with import_prefix, which adds prefix to go package paths for imported proto files. PS: Sorry for my English.

Reply:
This is v2, I'd like to change all of these `go_package`s to be correct anyway. Can we remove `import_prefix` if you update the `go_package`?

Reply:
no problem